src/content/reference/react/useDeferredValue.md

```diff
@@ -74,25 +74,25 @@ function SearchPage() {
 }
 ```
 
-During the initial render, the <CodeStep step={2}>deferred value</CodeStep> will be the same as the <CodeStep step={1}>value</CodeStep> you provided.
+Selama render awal, <CodeStep step={2}>nilai yang ditangguhkan</CodeStep> akan sama dengan <CodeStep step={1}>nilai</CodeStep> yang Anda berikan.
 
-During updates, the <CodeStep step={2}>deferred value</CodeStep> will "lag behind" the latest <CodeStep step={1}>value</CodeStep>. In particular, React will first re-render *without* updating the deferred value, and then try to re-render with the newly received value in background.
+Selama pembaruan, <CodeStep step={2}>nilai yang ditangguhkan</CodeStep> akan "tertinggal" dari <CodeStep step={1}>nilai</CodeStep> terbaru. Secara khusus, React pertama-tama akan re-render *tanpa* memperbarui nilai yang ditangguhkan, dan kemudian mencoba re-render dengan nilai yang baru diterima di latar belakang.
 
-**Let's walk through an example to see when this is useful.**
+**Mari telusuri contoh untuk melihat kapan ini berguna.**
 
 <Note>
 
-This example assumes you use one of Suspense-enabled data sources:
+Contoh ini menganggap Anda menggunakan salah satu sumber data Suspense-enabled:
```

Comment:
This needs to be confirmed by the others @resir014 @mazipan @r17x @zainfathoni, which one is better:

- Suspense-enabled
- yang memiliki Suspense
- yang terdapat Suspense
- yang diaktifkan dengan Suspense

Reply:
Need help to confirm which one is better to use @zainfathoni @resir014 @mazipan @r17x

Reply:
I think "yang menggunakan Suspense" is more suitable.

Reply:
Thanks @zainfathoni , delivered on this commit https://github.com/reactjs/id.react.dev/pull/360/commits/1f97fe3c470661b5ad30035066c287e28355bca2